Userland/Utilities/groupdel.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2020, Fei Wu <f.eiwu@yahoo.com>
+ * Copyright (c) 2021, Brandon Pruitt <brapru@pm.me>
+ * Copyright (c) 2021, Maxime Friess <M4x1me@pm.me>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ScopeGuard.h>
+#include <AK/String.h>
+#include <AK/StringBuilder.h>
+#include <LibCore/Account.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <ctype.h>
+#include <dirent.h>
+#include <errno.h>
+#include <grp.h>
+#include <pwd.h>
+#include <shadow.h>
+#include <spawn.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
```

Comment:
Some of those includes are unnecessary, e.g.:

```
#include <dirent.h>
#include <shadow.h>
#include <spawn.h>
#include <sys/wait.h>
```

I haven't checked all of them though.